src/schema.ts

```diff
@@ -1,10 +1,23 @@
 import { WithId } from 'mongodb';
 import types, { ObjectType } from './types';
-import { TimestampSchema, VALIDATION_ACTIONS, VALIDATION_LEVEL } from './utils';
+import {
+  TimestampSchema,
+  RequireAtLeastOne,
+  VALIDATION_ACTIONS,
+  VALIDATION_LEVEL,
+  getTimestampProperty,
+} from './utils';
 
-interface SchemaOptions<TProperties> {
+export type SchemaTimestampOptions =
+  | boolean
+  | RequireAtLeastOne<{
+      createdAt?: string;
+      updatedAt?: string;
+    }>;
+
+export interface SchemaOptions<TProperties> {
```

Comment:
Can we please export this type in the `index.ts`?

This is the first issue we ran into while trying this PR in our repo.

Reply:
Done!